docker-compose.yml

```diff
@@ -65,12 +65,12 @@ services:
     command: "full_local"
     environment:
       - ETHEREUM_NETWORK=LOCALCHAIN
-      - LOCALCHAIN_AUTHORITY_ADDRESS=0xc0f780dfc35075979b0def588d999225b7ecc56f
-      - LOCALCHAIN_TXHASH_CONTRACT=0xc47317b0de4c6ccc9a9fb858ad3ee858c2ee8ade3aed0fa9bcab7f7dad27fe12
-      - LOCALCHAIN_CONTRACT_ADDRESS_PLASMA_FRAMEWORK=0xc673e4ffcb8464faff908a6804fe0e635af0ea2f
-      - LOCALCHAIN_CONTRACT_ADDRESS_ETH_VAULT=0x0433420dee34412b5bf1e29fbf988ad037cc5db7
-      - LOCALCHAIN_CONTRACT_ADDRESS_ERC20_VAULT=0x04badc20426bc146453c5b879417b25029fa6c73
-      - LOCALCHAIN_CONTRACT_ADDRESS_PAYMENT_EXIT_GAME=0x1d92a9bef49bd340c7a11da536fb3ac9efbb1806
+      - AUTHORITY_ADDRESS=0xc0f780dfc35075979b0def588d999225b7ecc56f
```

Comment:
heads up: when you rebase on `master`, this is gone and you'll only make the change once in a `.env` file added next to this `.yml` file.

Reply:
Thanks

Reply:
oh, wait, I think I maybe miswrote. `localchain_contract_addresses.env` should be enough to be modified, no need to readd those values here, see https://github.com/omisego/elixir-omg/pull/1216/files#diff-4e5e90c6228fd48698d074241c2ba760R67